cloud-nio/cloud-nio-impl-drs/src/test/scala/cloud/nio/impl/drs/DrsCloudNioFileSystemProviderSpec.scala

```diff
@@ -0,0 +1,22 @@
+package cloud.nio.impl.drs
+
+import org.scalatest.matchers.should.Matchers
+
+class DrsCloudNioFileSystemProviderSpec extends org.scalatest.flatspec.AnyFlatSpec with Matchers {
+
+  behavior of "DrsCloudNioFileSystemProvider"
+
+  it should "not list paths" in {
```

Comment:
TOL is "not list paths" correct for what this is testing?

Reply:
... and from above it looks like it's true that paths shouldn't be listed, should there be a test for that here?

Reply:
Whoops, thanks! Will fix the test name.

The listing negative test is over here:
https://github.com/broadinstitute/cromwell/blob/d536faf00ac5290e5d03a7361377126d9e71b95a/cloud-nio/cloud-nio-impl-drs/src/test/scala/cloud/nio/impl/drs/DrsCloudNioFileProviderSpec.scala#L158-L160